apps/style/common.scss

```diff
@@ -1201,6 +1201,22 @@ $workspace-header-button-margin: ($workspace-headers-height - $workspace-header-
   }
 }
 
+.show-code {
+  summary {
+    font-size: 18px;
+    line-height: 20px;
+    color: #5b6770;
+    font-family: "Gotham 5r", sans-serif;
+    font-weight: normal;
+    outline: none;
+    padding: 5px;
+  }
+
+  .generated-code-container {
+    margin: 25px 10px 25px -80px;
+  }
+}
+
```

Comment:
it's less a PITA, and more a situation where I'm not convinced that the advantages of tightly coupled style+layout outweigh the advantages of loosely coupled style+layout; losing things like our `$color` and `$font` variables (which I'm not using here but really should be) and an already-existing framework for situational overrides.

On the other hand, if we keep some styles for our React stuff in CSS and use inline styles for some other react stuff, then it becomes even harder to track down what the style is.